### ISSUE TITLE
Notary on AARCH64

### ISSUE THREAD

**Issue by ShreyaGoel08** [docker/notary-official-images#7]
I am trying to build notary on AARCH64.
I have successfully built the binaries for notary-server and notary-signer on AARCH64. 
The docker is running succesfully if using Debian:stretch
```
FROM debian:stretch
EXPOSE 4443
COPY ./server-config.json /notary/server/
COPY ./notary-server /notary/server/
COPY ./entrypoint.sh /notary/server/
WORKDIR /notary/server
RUN useradd -ms \bin\sh notary
USER notary
ENV PATH=$PATH:/notary/server
ENTRYPOINT [ "entrypoint.sh" ]
CMD [ "notary-server", "--help" ]
```

But, I am geting error on using Alpine: latest as:
```
FROM alpine:latest
EXPOSE 4443
COPY ./server-config.json /notary/server/
COPY ./notary-server /notary/server/
COPY ./entrypoint.sh /notary/server/
WORKDIR /notary/server
RUN adduser -D -H -g "" notary
USER notary
ENV PATH=$PATH:/notary/server
ENTRYPOINT [ "entrypoint.sh" ]
CMD [ "notary-server", "--help" ]
```

To run the docker I am using the command
```docker run -ti --entrypoint=/bin/sh notary-server -s```

Please suggest what can be done to resolve the error arising in ```Alpine``` usecase.
Can i use ```Debian: Stretch``` for Notary?

Note: **In both the cases, I am using the same binaries,**

**Comment by justincormack** [comment]
What is the error you are getting?

**Comment by ShreyaGoel08** [comment]
I am getting the following error when running the docker 
```
[root@localhost notary-server]# docker run -ti --entrypoint=/bin/sh  notary-server -s
/notary/server $ notary-server -v
/bin/sh: notary-server: not found
/notary/server $
```
When I check, I can see notary-server is present there.
If using Debian: stretch then I am not receiving the error.

**Comment by justincormack** [comment]
That means the build is dynamically linked against glibc.

**Comment by ShreyaGoel08** [comment]
Can you please guide as what should be done to execute it.
Or should I use Debian: stretch.

**Comment by ShreyaGoel08** [comment]
Thanks for helping us out.
After using the static binaries the issues got resolved for me.

To support on AARCH64(ARM64v8) Do I need to upload the binaries for notary-server and notary-signer.

**Comment by ShreyaGoel08** [comment]
Thanks for helping us out.
After using the static binaries the issues got resolved for me.

**To support on AARCH64(ARM64v8) Do I need to upload the binaries for notary-server and notary-signer.**

**Comment by odidev** [comment]
Hi,
I have raised a **PR** at #9, so as to provide the way I did it for ```arm64 & amd64```
These changes will generate Notary binaries for AMD64 and ARM64v8 as well.

Regards,

**Comment by lag-linaro** [comment]
+1 for this.

Please consider it for merge.

**Comment by lag-linaro** [comment]
This should be fixed by #10

**Comment by lag-linaro** [comment]
\o/
